src/openforms/submissions/api/viewsets.py

```diff
@@ -126,6 +126,13 @@ class SubmissionViewSet(
     lookup_field = "uuid"
     pagination_class = pagination.PageNumberPagination
 
+    def get_throttles(self):
+        if self.action == "_complete":
+            self.throttle_scope = "submit"
```

Comment:
is this what DRF recommends to set the throttle scope? looks a bit weird and not-best practices to me

Reply:
According to the [DRF docs](https://www.django-rest-framework.org/api-guide/viewsets/#introspecting-viewset-actions)

Reply:
that's about introspecting, not about setting the `trottle_scope`. However, I could figure out from the source code that `view.throttle_scope` is checked (if it exists), so a cleaner implementation would be:

```py
@property
def throttle_scope(self):
    # called/checked by the DRF scoped throttle class
    if self.action == "_complete":
        return "submit"
    if self.action == "_suspend":
        return "pause"
    return None
```

magically setting attributes on `self` is discouraged, as it's hard to track down where they come from and where they are used.